roles/teamcomments/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Checkout TeamComments
```

Comment:
Not saying anything more about hills.  Maybe I'm just receiving medical treatment on them, and there is still hope.

Reply:
I really tempted to leave this one, just because you were less committed to it than the others.